server/migrations/20190319030801-create-comment.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Comments', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      body: {
+        type: Sequelize.STRING
```

Comment:
**[comment]** Please verify that the column type here can fit the data that will be written to it

Reply:
I have updated the data type to text. This is because text data type allows up to 65,535 characters unlike using string.

https://sequelize.readthedocs.io/en/2.0/api/datatypes/
https://stackoverflow.com/questions/6766781/maximum-length-for-mysql-type-text